api/build.gradle

```diff
@@ -39,15 +45,15 @@ plugins {
     id 'io.spring.dependency-management' version '1.0.11.RELEASE'
     id 'com.diffplug.spotless' version '5.12.5'
     id 'com.google.cloud.tools.appengine-appenginewebxml' version '2.4.5'
-    id 'org.jetbrains.kotlin.jvm' version '1.5.32'
+    id 'org.jetbrains.kotlin.jvm' version '1.8.20'
```

Comment:
is there a way for these to use the `ext` values from above?

Reply:
I wanted that too and tried several variants before finally giving up.